### ISSUE TITLE
Packages for next version

### ISSUE THREAD

**Issue by dannyparsons** [africanmathsinitiative/R-Instat#5976]
- stringdist

@rdstern @lilyclements @africanmathsinitiative/developers any new packages that need to be added for newly created dialogs?

**Comment by shadrackkibet** [comment]
- imputeTS
- chillR
-changepoint
-trend

**Comment by Patowhiz** [comment]
ggplotgui package  as mentioned in issue #5146 ?

**Comment by dannyparsons** [comment]
Is the dialog implemented yet? Let's only include packages once a dialog is merged so we avoid unnecessarily adding packages before they are in use.

**Comment by dannyparsons** [comment]
- pbs #6006

**Comment by dannyparsons** [comment]
- visdat

**Comment by dannyparsons** [comment]
@africanmathsinitiative/developers please keep track of PRs that you create which require new R packages to be added to the project. We have found a number of R packages used in active dialogs already merged which were not on this list or not linked to PRs. These dialogs won't work in the release version of R-Instat unless we know we need to add them to the list of R packages.

I suggest we have a procedure that if you create a new PR that requires a new R package, put a line in the description of the PR that states the new R packages. Then also add a comment to the open issue on "Packages for next version". I will try to make sure there is always an open issue for this. When adding to this list please also link to the PR as Shadrack has done so that we know when the PR is merged. Thanks.
